response/response.go

```diff
@@ -28,6 +28,12 @@ func New(body interface{}) *Response {
 }
 
 func (res *Response) Status(code int) *Response {
+	if !validStatusCode[code] {
```

Comment:
I don't think we should check status code is valid or invalid, maybe the user wants to use the custom status code.

Reply:
But HTTP status codes are extensible.
From RFC 2616 https://tools.ietf.org/html/rfc2616#section-6.1.1

Reply:
> The Status-Code element is a 3-digit integer

According to this, I think we should check it's `>= 100 && <= 999`?

Reply:
Yes, I think it will be great.